### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by albertoloscerritos** [SilentVoid13/Templater#1166] [Help]
Just want to know, thanks.

**Comment by Zachatoo** [comment]
I think it depends on how you define "dead."

It hasn't gotten updates in a while, and I'm not expecting it to get any updates for a while.

But it still works, and the Templater community is fairly active here and in the Obsidian Discord.

**Reply by albertoloscerritos** [reply]
Yes, my question was primarily from a point of view of new updates. And a better compatibility with the new 1.4.0. frontmatter. 
Hope we can see some new updates in the future. Thanks for your answer.

**Reply by Zachatoo** [reply]
Is there something specific that isn't working with 1.4.0? I'm on 1.4.3 and haven't had any issues with Templater and properties.

**Reply by huyz** [reply]
I'm also worried about the status of this project.  Version 1.4's introduction of properties are conflicting with Templater's handling of frontmatter.  I downgraded immediately when that happened.  Now that v1.4 is out for general release, I'll try again, but with no updates from Templater, I'm wary.

**Reply by AB1908** [reply]
Can someone create an issue with repro instructions? I can try to get a hotfix out over the weekend.

**Reply by huyz** [reply]
Ok I just installed the latest Obsidian v1.4.5. And it seems that the core devs fixed whatever was broken!

Will report back in a day or two.
